buildrunner/docker/multiplatform_image_builder.py

```diff
@@ -310,7 +316,8 @@ def _build_single_image(
             f"'{file}'({os.path.exists(f'{file}')}) does not exist!"
 
         tagged_names = [f"{name}:{tag}" for tag in tags]
-        logger.debug(f"Building {tagged_names} for {platform}")
+        builder = self._platform_builders.get(platform) if self._platform_builders else None
+        logger.debug(f"Building {tagged_names} for {platform} with {builder or 'default'} builder")
```

Comment:
I didn't know about this syntax. Cool.

Reply:
It's not as typical and I don't think it's as recommended honestly, though it is shorter.